docs/readme.md

```diff
@@ -1,37 +1,24 @@
 # AutoRest Documentation
 
-1. Installing AutoRest. 
-    - You can [install AutoRest](./installing-autorest.md) on Windows, Mac, or Linux.
+1. Introduction
+    - Get an [introduction](./introduction.md) to the fundamental concepts of AutoRest
 
-1. [Defining Clients With OpenAPI](developer/guide/how-autorest-generates-code-from-swagger.md)
-    - AutoRest-specific [OpenAPI Extensions](extensions/readme.md)
+2. Installing AutoRest
+    - You can [install AutoRest](./install/readme.md) on Windows, Mac, or Linux.
 
-2. Generating Clients
-    - [Command Line Interface](user/command-line-interface.md) - using AutoRest from the command line.
```

Comment:
Should this file be removed as well? I just updated it, with some additional info for the Github token flag which I don't see referenced anywhere else

Reply:
added this info to the main generate/readme.md. thanks!